src/Tribe/Updater.php

```diff
@@ -50,10 +50,44 @@ public function do_updates() {
 		}
 	}
 
+	/**
+	 * Function fired when we evaluate a plugin version bump to be tracked in the database.
+	 *
+	 * @param string $new_version The new version for this plugin.
+	 */
 	public function update_version_option( $new_version ) {
+		/**
+		 * Hook that will only fire when a plugin version has changed, enabling some one-off actions to be performed.
+		 * Important when using this hook to realize this hook will fire for various plugins, and the combination of
+		 * both params are required in order to target a particular action for a particular version on a particular plugin.
+		 *
+		 * This is fired immediately before the version change.
+		 *
+		 * @since TBD
+		 *
+		 * @param string $version_option  The version option being used for this plugin. This is necessary to know which plugin is updating.
+		 * @param string $new_version     The new version number.
+		 * @param string $current_version The current version number.
+		 */
+		do_action( 'tec_events_plugin_updater_version_change',
```

Comment:
What about adding the version option to the action name? 

Or adding a 2nd hook that does that?

So VE would have tec_events_plugin_updater_version_change_events-virtual-schema-version|
ET+ : event-tickets-plus-schema-version

That would make it a it easier to target specific plugins.